docs/developer-docs/latest/development/backend-customization/models.md

```diff
@@ -104,8 +104,9 @@ Many types of attributes are available:
 
 - scalar types (e.g. strings, dates, numbers, booleans, etc.),
 - Strapi-specific types, such as:
-  - `media`, for files uploaded through the [Media library](/user-docs/latest/content-types-builder/configuring-fields-content-type.md#media)
+  - `media` for files uploaded through the [Media library](/user-docs/latest/content-types-builder/configuring-fields-content-type.md#media)
   - `relation` to describe a [relation](#relations) between content-types
+  - `customFields` to describe [custom fields](#custom-fields) and their specific keys
```

Comment:
Should this be customField? Since `type: customField`

Reply:
Indeed, good catch, thanks!